cypress.config.ts

```diff
@@ -0,0 +1,26 @@
+import { defineConfig } from 'cypress';
+import resetDatabase from '@test/helpers/database.reset';
+import { createUser } from '@test/helpers/database.seed';
+
+export default defineConfig({
+  e2e: {
+    baseUrl: 'http://localhost:3000',
+    setupNodeEvents(on, config) {
+      on('task', {
+        async resetAll() {
```

Comment:
Adding the `All` brings more confusion as it could mean also resetting mocks or env data.

So the following would be more descriptive:

```suggestion
        async resetDatabase() {
          return resetAllDatabaseTables();
        }
```